suites/portal/dataUploadTest.js

```diff
@@ -106,3 +108,13 @@ AfterSuite(async (sheepdog, indexd, files, dataUpload) => {
     files.deleteFile(fileName);
   });
 });
+
+Scenario('Upload clinical data file through portal large enough to trigger chunking @manual @PXP-5053', ifInteractive(
+  async () => {
+    const result = await interactive(`
+        1. Submit a clinical data file with enough lines to trigger chunking (>30 rows in the default setting)
+        2. Data is submitted successfully and the state is shown as Succeeded in the 'Recent Submissions' pane
+      `);
+    expect(result.didPass, result.details).to.be.true;
+  },
```

Comment:
This is a great test!
Would you add a `dataUploadTestCases.tsv` or something like that - just a sample file that someone can start with, and massage to fit the commons they're testing against?  Or maybe give instructions for how to get the data-simulator to generate a file for you?

Reply:
Added the files I used to test